packages/cache/__tests__/cacheUtils.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as io from '@actions/io'
+import {promises as fs} from 'fs'
+import * as path from 'path'
+import * as cacheUtils from '../src/internal/cacheUtils'
+
+jest.mock('@actions/core')
+jest.mock('os')
+
+function getTempDir(): string {
+  return path.join(__dirname, '_temp', 'cacheUtils')
+}
+
+afterAll(async () => {
```

Comment:
Are these needed? I don't think there is anything created in the TempDir with these 2 tests. With `fs.mkdtemp` that is used below, I think it uses some other directory